dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNServiceManagerInterfaceIIDM.cpp

```diff
@@ -62,7 +62,8 @@ ServiceManagerInterfaceIIDM::getBusesConnectedBySwitch(const std::string& busId,
   }
 
   if ((*vlIt)->getVoltageLevelTopologyKind() == VoltageLevelInterface::NODE_BREAKER) {
-    throw DYNError(Error::MODELER, VoltageLevelTopoError, VLId);
+    // Do not throw error
+    Trace::warn() << "Node/breaker support in progress (#1458)" << Trace::endline;
```

Comment:
We generally avoid referencing issue tickets in the comments

Reply:
Thinking about it, please leave the error in the master of dynawo until the feature is ready

Reply:
So we keep the `Trace::warn` but remove the issue number from the message ?

Reply:
I propose to revert this change and to commit it when the work on node breaker support will be finished, would it be ok with you?

Reply:
I have removed the reference to the issue in the code. I have left the message at the debug level.

Reply:
Sorry if I wasn't clear: we do not want to have an unfinished algorithm accessible in the master. Please revert this modification and keep it only on your branch

Reply:
I am sorry, but I am not sure to completely understand. 

With the changes made in this pull request, we have verified that node/breaker topologies can be handled by Dynawo without problems. We have verified that the solutions of DynaFlow for the bus/branch and node/breaker versions of an IEEE118 derived test case are identical, and we have also checked it against a real-world example. 

As far as I know, the node/breaker topologies will be supported by Dynawo with this set of changes. Is there anything else that I am missing ? What is the unfinished algorithm from your point of view ?

I made an additional commit changing the message in this line, and lowering the trace level to `debug`.

Reply:
I would be ok if we increase the unit test `TEST(DataInterfaceTest, ServiceManager)` in  `Modeler/DataInterface/PowSyblIIDM/test/TestServiceManagerInterface.cpp` to include some checks about `vlIIDM2` that is created as `NODE_BREAKER` ? This will verify that the implementation of `getBusesConnectedBySwitch` is correct and works for node/breaker.

Reply:
Regarding the warning if the algorithm was tested and validated then I guess that we don't need the warning anymore, so maybe we can just remove it entirely.
I totally agree with the adding a new unit test for this usage.

Reply:
Modified the mentioned unit test to include a node/breaker configuration. The two voltage levels created in the test have similar connections, one is bus/breaker and the other node/breaker, including internal connections.